routes/celebrities.routes.js

```diff
@@ -0,0 +1,27 @@
+const router = require("express").Router();
+const Celebrity = require("../models/Celebrity.model");
+
+// all your routes here
+router.get("/", (req, res, next) => {
+  Celebrity.find()
+    .then((Celebrities) => {
```

Comment:
este "Celebrities" está en mayúsculas y el que le pasáis a la vista en minúsculas "celebrities" por lo que la vista falla ya que no le llegan datos